web-marketplace/src/components/organisms/SellOrdersActionsBar/SellOrdersActionsBar.tsx

```diff
@@ -76,18 +95,33 @@ export const SellOrdersActionsBar = ({
         }}
       >
         {isAdmin ? (
```

Comment:
right now, `isAdmin` would always be false if connected with wallet connect/keplr mobile web, see how it's declared: https://github.com/regen-network/regen-web/blob/b9220b5ed159a3d667337b36d082cf5ecae43e7f/web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx#L361-L364 (since in this case, there's no `activeAccount`)

That means that instead we show the regular action bar with "buy" button.
I've discussed this with @erikalogie and instead, we should just hide the action bar so we should update `isAdmin` declaration adding a check for `wallet.address && wallet.address === onChainProject?.admin === activeAccount?.addr` and then if `loginDisabled` and `isAdmin` we hide the action bar.

Reply:
@blushi I'm not sure the expression you've written would work as expected. You mean something like: `  const isAdmin =
    (!!activeAccount?.addr && onChainProject?.admin === activeAccount?.addr) ||
    (!!activeAccount?.id &&
      offChainProject?.adminAccountId === activeAccount?.id) ||
    (wallet?.address && wallet?.address === onChainProject?.admin) ||
    (wallet?.address && wallet?.address === activeAccount?.addr);`

Reply:
@blushi please see whole updated code in PR instead of just my previous message

Reply:
yes sorry I've updated my comment, this would give

```
const isAdmin =
        (!!activeAccount?.addr && onChainProject?.admin === activeAccount?.addr) ||
        (!!activeAccount?.id &&
          offChainProject?.adminAccountId === activeAccount?.id) ||
        (loginDisabled && wallet?.address && wallet?.address === onChainProject?.admin)
```

After second thought, it's probably safer to add `loginDisabled` check as well in there
because it's possible to have a logged in user `activeAccount` with an address `activeAccount.addr` that is different from `wallet.address` (for instance, if the user logged in with email but a different wallet than the one associated to his account is selected in Keplr) and this particular case, we would not want `isAdmin` to be true for a project that the account in Keplr is an admin of.

Reply:
This is now updated accordingly. Please review PR when you get a chance.